bin/MadGraph5_aMCatNLO/cards/ggh01_Toa01a01_Tomumutautau/haa_h125_10_gp/ggh01_M125_Toa01a01_M10_Tomumutautau_run_card.dat

```diff
@@ -0,0 +1,285 @@
+    -1.0=  ht4max  ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+    0.0=  ptgmin  ! Min photon transverse momentum
+    0.4=  R0gamma  ! Radius of isolation code
+    1.0=  xn  ! n parameter of eq.(3.4) in hep-ph/9801442
+    1.0=  epsgamma  ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+    True=  isoEM  ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+    0.0=  xetamin  ! minimum rapidity for two jets in the WBF case  
+    0.0=  deltaeta  ! minimum rapidity for two jets in the WBF case 
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+    -1.0=   ktdurham        
+    0.4=   dparameter 
+ #*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+    4=  maxjetflavor     ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+    0.0=  xqcut    ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+    True=  use_syst       ! Enable systematics studies
+#
+#**************************************
+# Parameter of the systematics study
+#  will be used by SysCalc (if installed)
+#**************************************                                  
+#
+#********************************************************************* 
+#  Additional parameter
+#*********************************************************************
+    1.0= ptonium # hidden parameter
+    0.6= etaonium # hidden parameter
```

Comment:
Please remove the lines below True = use_syst.
This might cause problems when parsing weights in NanoAOD maybe